ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/console/graph/GraphPanelInteractions.java

```diff
@@ -67,6 +69,30 @@ public void handleError(Exception exception) {
                     public void executionCompleted() {
                     }
                 });
+
+        messageBus.connect()
+                .subscribe(QueryExecutionProcessEvent.QUERY_EXECUTION_PROCESS_TOPIC, new QueryExecutionProcessEvent() {
+                    @Override
+                    public void executionStarted(ExecuteQueryPayload payload) {
+                    }
+
+                    @Override
+                    public void resultReceived(GraphQueryResult result) {
+                    }
+
+                    @Override
+                    public void postResultReceived() {
+                    }
+
+                    @Override
+                    public void handleError(Exception exception, Editor editor) {
```

Comment:
Well. While everything is OK... Let's be real boyscouts and clean up here a bit.

Let's do this:
- Add `ExecuteQueryPayload payload` as a first argument to all methods here. It's my design failure that I did not did it. While we don't need them everywhere right now, it's better to have them.
- Move `editor` inside `ExecuteQueryPayload`. It's not great idea to expand number of parameters for no real reason.